Changes

```diff
@@ -161,6 +161,9 @@ Working version
 
 ### Bug fixes:
 
+- #7156: Create Mechanism so that OCaml Top Level uses custom printers 
```

Comment:
Trailing space on this line (see details of the hook in [CONTRIBUTING.md](https://github.com/ocaml/ocaml/blob/trunk/CONTRIBUTING.md#coding-guidelines))